File: persephone/provisioning-service/src/main/java/com/vmware/blockchain/deployment/services/orchestration/model/vsphere/LibraryItemFindSpec.java
/*
 * Copyright (c) 2020 VMware, Inc. All rights reserved. VMware Confidential
 */

package com.vmware.blockchain.deployment.services.orchestration.model.vsphere;

import lombok.Builder;
import lombok.Data;

/**
 * Data class.
 */
@Data
@Builder
public class LibraryItemFindSpec {
    String name;
    String libraryId;
    String sourceId;
    String type;
    Boolean cached;
}

File: persephone/orchestration-vmware/src/common/com/vmware/blockchain/deployment/model/vsphere/LibraryItemFindSpec.kt
/* **************************************************************************
 * Copyright (c) 2019 VMware, Inc.  All rights reserved. VMware Confidential
 * *************************************************************************/
package com.vmware.blockchain.deployment.model.vsphere

import kotlinx.serialization.Serializable

@Serializable
data class LibraryItemFindSpec(
    val name: String? = null,
    val library_id: String? = null,
    val source_id: String? = null,
    val type: String? = null,
    val cached: Boolean? = null
)
